spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -1020,6 +1020,30 @@ the request enters the JAX-RS method and the timeout information found on the `@
 the moment calculated for the LRA at that time, this new moment becomes the time out moment for the LRA.
 So when multiple participants define a timeout period, the earliest one will trigger the cancellation of the LRA.
 
+Since the specification targets microservices running in different JVM's there are
+potential issues around how timing constraints are handled:
+
+A timeout is a duration which results in some calculated end time. If different systems
+use different timezone or clock granularity then there is potential for different views
+about the current time.  Therefore implementations must be careful to avoid using absolute
+times and be aware that absolute synchronisation between clocks is not possible, and
+therefore, where possible, rely on a duration instead. If an absolute time is required
+then the recommendation is to use UTC which is a universal time standard to which
+all timezones can be canonically converted. This will provide some degree of alignment
```

Comment:
I may be missing the idea in this paragraph but we allow only durations in LRA annotation. Can you remind me why do we need to describe absolute times, please?

Reply:
> I may be missing the idea in this paragraph but we allow only durations in LRA annotation. Can you remind me why do we need to describe absolute times, please?

When a crashed server is restarted the implementation must calculate when timed LRAs become eligible for cancellation.

Reply:
With `crashed server` do you mean the implementation or the micro-service?

For me, the 'time-out' time is calculated at the moment a participant joins and monitored all the time to see if LRAs become eligible for cancellation. (and callbacks needs to be sent out)

When another participant joins, the time-out needs to be verified and possible redefined (when participant defines also a duration which might be earlier then the current end time).

So I see a role for the UTC time there, which doesn't has anything to do with a restart of a crashed implementation or restart of micro-service.

Reply:
I believe this case is referring to a situation where all actors in the system are shutdown while the LRA is still active. In order that the system can rehydrate successfully it needs to know the expected time that the LRA was expected to automatically cancel.